app/views/items/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<section class="item-info">
+  <% @item.each do |item| %>
+    <p><%= "Item Name: #{item.name}" %></p>
+    <p><%= "Item Price: #{item.price}" %></p>
+    <p><%= "Supermarket: #{item.supermarket.name}" %></p>
+    <p><%= "Customer Purchase Count: #{item.purchase_number}" %></p>
```

Comment:
Good job wrapping this logic in a model method. 

Also, on line 5, you _could_ also create a model method of `supermarket_name` and have the logic in the model. It's a small change, but it adheres more to Law of Demeter and reduces digging into nested objects.